2021-01-11 16:12:49,405 INFO    MainThread:12732 [wandb_setup.py:_flush():70] setting env: {}
2021-01-11 16:12:49,405 INFO    MainThread:12732 [wandb_setup.py:_flush():70] multiprocessing start_methods=spawn
2021-01-11 16:12:49,406 INFO    MainThread:12732 [wandb_setup.py:_flush():70] setting login settings: {}
2021-01-11 16:12:49,406 INFO    MainThread:12732 [wandb_init.py:_log_setup():303] Logging user logs to C:\Users\kevin\OneDrive\Documents\8th sem\Federated_Pneumonia\src\client\wandb\run-20210111_161249-tvmllzqd\logs\debug.log
2021-01-11 16:12:49,406 INFO    MainThread:12732 [wandb_init.py:_log_setup():304] Logging internal logs to C:\Users\kevin\OneDrive\Documents\8th sem\Federated_Pneumonia\src\client\wandb\run-20210111_161249-tvmllzqd\logs\debug-internal.log
2021-01-11 16:12:52,030 INFO    MainThread:12732 [wandb_run.py:_console_start():1237] atexit reg
2021-01-11 16:12:52,032 INFO    MainThread:12732 [wandb_run.py:_redirect():1104] redirect: SettingsConsole.WRAP
2021-01-11 16:12:52,032 INFO    MainThread:12732 [wandb_run.py:_redirect():1135] Wrapping output streams.
2021-01-11 16:12:52,033 INFO    MainThread:12732 [wandb_run.py:_redirect():1151] Redirects installed.
2021-01-11 16:13:02,390 INFO    MainThread:12732 [wandb_run.py:_atexit_cleanup():1207] got exitcode: 1
2021-01-11 16:13:02,390 INFO    MainThread:12732 [wandb_run.py:_restore():1179] restore
2021-01-11 16:13:02,396 INFO    MainThread:12732 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "12737abc75394e129602270a77858d3f"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 938
      total_bytes: 938
    }
  }
}

2021-01-11 16:13:04,420 INFO    MainThread:12732 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "b175e3ddd2e04c93a2227290b83ced40"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 6305
      total_bytes: 6305
    }
  }
}

2021-01-11 16:13:05,527 INFO    MainThread:12732 [wandb_run.py:_show_files():1536] logging synced files
2021-01-11 16:13:05,533 INFO    MainThread:12732 [internal.py:handle_exit():138] Internal process exited
